app/routes/organen.js

```diff
@@ -0,0 +1,36 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import RSVP from 'rsvp';
+
+export default class OrganenRoute extends Route {
+  @service currentSession;
+  @service session;
+  @service router;
+  @service store;
+
+  beforeModel(transition) {
+    this.session.requireAuthentication(transition, 'login');
+
+    // TODO we probably want some access rights here.
+    // if (!this.currentSession.canAccessBestuursorganen)
+    //   this.router.transitionTo('index');
+  }
+
+  async model() {
+    const bestuurseenheid = this.currentSession.group;
+    const bestuursorganen = await this.getAllBestuursorganen(
```

Comment:
i think we should filter here and only get the bestuursorganen that aren't a 'tijdspecialisatieVan' of something else (i.e. not the bestuursorganen in de tijd)

Reply:
fine to do this in a separate pr though! this structure is great to have already!

Reply:
this also needs some extra filtering with regard to activeness, will do this in the next PR